File: src/main/java/me/celestialfault/celestialwynn/config/ModMenuImpl.java
package me.celestialfault.celestialwynn.config;

import com.terraformersmc.modmenu.api.ConfigScreenFactory;
import com.terraformersmc.modmenu.api.ModMenuApi;

public class ModMenuImpl implements ModMenuApi {
	@Override
	public ConfigScreenFactory<?> getModConfigScreenFactory() {
		return ConfigGUI::getConfigScreen;
	}
}

File: src/main/kotlin/dev/celestialfault/celestialwynn/config/ModMenuImpl.kt
package dev.celestialfault.celestialwynn.config

import com.terraformersmc.modmenu.api.ConfigScreenFactory
import com.terraformersmc.modmenu.api.ModMenuApi

class ModMenuImpl : ModMenuApi {
	override fun getModConfigScreenFactory() = ConfigScreenFactory(ConfigGUI::getConfigScreen)
}
